recipes/fgsl/meta.yaml

```diff
@@ -0,0 +1,58 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "fgsl" %}
+{% set version = "1.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/reinh-bader/fgsl/archive/{{ version }}.tar.gz
+  sha256: 5013b4e000e556daac8b3c83192adfe8f36ffdc91d1d4baf0b1cb3100260e664
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('fortran') }}
+    - make
+    - autoconf
+    - automake
+    - m4
+    - pkg-config
+    - libtool
+    - gettext
+  host:
+    - gsl =2.6
```

Comment:
can you use either `2.6` or `==2.6`?

Reply:
Addressed via https://github.com/conda-forge/staged-recipes/pull/16107/commits/f37be4a5ba7dbfc95d90f3820120946d54149e87